services/namex-pay/tests/docker-compose.yml

```diff
@@ -1,16 +1,18 @@
-
-version: '2.1'
+# local postgres
+version: '1'
 services:
-    nats:
-        image: nats-streaming 
-        restart: always
-        mem_limit: 512m
-        expose:
-          - 4222
-          - 8222
-        labels:
-          - entity.services=nats
-        ports:
-          - 4222:4222
-          - 8222:8222
-        tty: true
\ No newline at end of file
+  db:
+    image: postgres
+    container_name: namex-pay-testing
+    environment:
+      - POSTGRES_DB=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
```

Comment:
Going to have issues if you drop databases, it wont let you drop database `postgres`